api/views.py

```diff
@@ -286,6 +287,18 @@ class AssetSpecsViewSet(ModelViewSet):
     http_method_names = ['get', 'post', 'put']
 
 
+class GroupViewSet(ModelViewSet):
+    serializer_class = UserGroupSerializer
+    queryset = Group.objects.all()
+    permission_classes = [IsAuthenticated, IsAdminUser]
+    authentication_classes = (FirebaseTokenAuthentication,)
+    http_method_names = ['get', 'post']
+
+    def perform_create(self, serializer):
+        name = " ".join(serializer.validated_data.get('name').title().split())
```

Comment:
<img width="1651" alt="screen shot 2018-08-03 at 11 30 23" src="https://user-images.githubusercontent.com/10899666/43634315-6e3239cc-9714-11e8-8cac-093ddea81b4f.png">

Add a try catch statement to resolve this **IntegrityError**.